lib/licence_location_identifier.rb

```diff
@@ -1,30 +1,36 @@
-require 'location_identifier'
+class LicenceLocationIdentifier
+  AUTHORITY_TYPES = %w(DIS LBO UTA CTY LGD MTD COI).freeze
+  ALLOWED_REQUESTED_AREA_TYPES = %(CTY).freeze
+
+  def self.find_slug(areas, requested_area_type = "")
```

Comment:
Wondering whether it would be better to have `county_requested` here as the second parameter (instead of `requested_area_type`). Otherwise any class calling this one would have to know about MapIt style area descriptions (well only "CTY")...